docs/admin_api/purge_history_api.md

```diff
@@ -13,65 +12,67 @@ delete the last message in a room.
 
 The API is:
 
-``POST /_synapse/admin/v1/purge_history/<room_id>[/<event_id>]``
+```
+POST /_synapse/admin/v1/purge_history/<room_id>[/<event_id>]
+```
 
-To use it, you will need to authenticate by providing an ``access_token`` for a
-server admin: see `README.rst <README.rst>`_.
+To use it, you will need to authenticate by providing an `access_token` for a
+server admin: see [Admin API](../../usage/administration/admin_api/index.html).
```

Comment:
is this link expected to work? It's a 404 right now.

Reply:
likewise a bunch of similar links below.

Reply:
Ah, there's two things here:

* That link only works in the context of https://github.com/matrix-org/synapse/pull/10086 which this commit was pulled out of.
* We shouldn't link to `...index.html` as that won't work when viewing the docs on GitHub. It should have been `../../usage/administration/admin_api/README.md`. I'll keep this in mind when updating https://github.com/matrix-org/synapse/pull/10086 once this has merged.